test/ui-test/src/screens/SettingsScreen.py

```diff
@@ -28,6 +28,9 @@ class WalletSettingsScreen(Enum):
     TWELVE_SEED_PHRASE: str = "twelve_seed_phrase_address"
     EIGHTEEN_SEED_PHRASE: str = "eighteen_seed_phrase_address"
     TWENTY_FOUR_SEED_PHRASE: str = "twenty_four_seed_phrase_address"
+    GENERATED_ACCOUNTS: str = "settings_Wallet_MainView_GeneratedAccounts"
+    DELETE_ACCOUNT: str = "settings_Wallet_AccountView_DeleteAccount"
```

Comment:
I'm wondering if we should have a naming convention so, if it is a button, finish with `_BTN`, if it is an input, finish with `_INPUT` or something similar to have a better vision when you read it in code. WDYT?